### DISCUSSION TITLE
Update Auto to 1.9.3

### DISCUSSION THREAD

**Discussion by SachiaLanlus** [AbdBarho/stable-diffusion-webui-docker#672] [General]
Update Auto to 1.8.0.
This update come with FP8 support which is important to me and I think also someone who using unexpensive GPU.
All update details can be find [here](https://github.com/AUTOMATIC1111/stable-diffusion-webui/releases/tag/v1.8.0).

**Comment by Krisseck** [comment]
1.9.0 was just released as well https://github.com/AUTOMATIC1111/stable-diffusion-webui/releases/tag/v1.9.0

**Comment by simonmcnair** [comment]
If you want a more up to date version of A1111 here are the steps:

docker exec -it webui-docker-auto-1 /bin/bash
git pull
pip install -r requirements.txt
#if needed
cd repositories
git clone https://github.com/AUTOMATIC1111/stable-diffusion-webui-assets.git
exit

docker stop webui-docker-auto-1
docker start webui-docker-auto-1

If it fails for any reason just rebuild the container using the standard process of docker compose --profile auto up --build

**Reply by manic101** [reply]
This also works to update to 1.10.1 too
However i had to change `git clone` to `git fetch`

**Comment by SachiaLanlus** [comment]
v1.9.0 is more like a chore fix.
just replace "v1.8.0" in Dockerfile with "v1.9.0" to update
